www/web-proxy-useracl/src/opnsense/scripts/OPNsense/ProxyUserACL/reconfigure.php

```diff
@@ -37,9 +37,9 @@
 $mdlProxyUserACL = new ProxyUserACL();
 $domain = strtoupper((string) Config::getInstance()->object()->system->domain);
 
-array_map('unlink', glob("/usr/local/etc/squid/ACL_*.txt"));
+array_map('unlink', glob("/usr/local/etc/squid/ACL_useracl_*.txt"));
```

Comment:
this should be removed as a long term goal is to run php as a user which means that it will probably not be able to write this file.

Reply:
So such reconfigure scripts have to be shell-scripts only?

Reply:
@evbevz all actions which run as root cannot be done in PHP in the future and therefore must be implemented in some SUID / configd call which does the job. For now it works because the legacy code needs root permission but when the sections are extracted, your plugin will be broken.

Reply:
Where to move the generated ACLs ?